packages/outline-core-link/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@phase2/outline-core-link",
+  "version": "0.1",
```

Comment:
This should be `0.0.1` to start. I'm changing this in my local copy and will push an update to test build.